acbu_minting/src/lib.rs

```
@@ -1,1296 +1,2 @@
-
-    for &b in slice.iter() {
-        let valid = matches!(b,
-            b'A'..=b'Z' | b'a'..=b'z' | b'0'..=b'9' | b'-' | b'_'
-        );
-        if !valid {
-            env.panic_with_error(MintingError::FintechTxIdInvalidChar);
-        }
-    }
-    let _ = env; // env kept in signature for future on-chain logging
-}
+use soroban_sdk::{contract, contractimpl, contracttype, symbol_short, Address, BytesN, Env, String};
```

Clippy (1.95.0)

[warning] unused imports

(warning)